web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -317,6 +322,7 @@ export const DailyViewTable = (props: Props) => {
                               <TableCell className={classes.planningArea} colSpan={19}>
                                 {area.name}
                               </TableCell>
+                              <TableCell>{/** Spacing cell */}</TableCell>
```

Comment:
you could delete this empty TableCell and instead just increase the `colSpan` of the previous TableCell to `20`